frontend/src/_styles/drawer.scss

```diff
@@ -62,14 +62,15 @@
   visibility: hidden;
   opacity: 0;
   background: rgba(0, 0, 0, 0.5);
-  transition: opacity var(--transition-speed) ease, visibility var(--transition-speed) ease;
   width: 100%;
   height: 100%;
   top: 0;
   left: 0;
   position: fixed;
   pointer-events: none;
   z-index: 0;
```

Comment:
This will affect all the drawer transition right. Should we add it specific for ToolJet db